unstructured_ingest/v2/processes/connectors/astradb.py

```diff
@@ -31,6 +43,38 @@
 CONNECTOR_TYPE = "astradb"
 
 
+def get_astra_collection(connection_config, config) -> "AstraDBCollection":
```

Comment:
Can the connection config class have a method that returns just the client: 
```python
    def get_client(self) -> "AstraDBClient":
        from astrapy import DataAPIClient as AstraDBClient

        # Create a client object to interact with the Astra DB
        # caller_name/version for Astra DB tracking
        return AstraDBClient(
            caller_name=integration_name,
            caller_version=integration_version,
        )
```

This matches the pattern elsewhere. 

Or if we move some of the fields, it seems that the database connection is required as part of the connection, regardless of download or upload, so some of those can be moved to the connection config and have a method that return the database object:

```python
    def get_database(self) -> "AstraDBDatabase":
        from astrapy import DataAPIClient as AstraDBClient

        # Create a client object to interact with the Astra DB
        # caller_name/version for Astra DB tracking
        client =  AstraDBClient(
            caller_name=integration_name,
            caller_version=integration_version,
        )

        return client.get_database(
            api_endpoint=self.api_endpoint,
            token=self.token,
            keyspace=self.keyspace,
        )
```

Reply:
I added a get_client method in the connection config class. I wasn't able to add a database/collection getter in the connection config as well, the info needed is in (and unique to?) the Indexer/Downloader/Uploader configs
Since the existing get_astra_collection is used for both the indexer and downloader, I left that method outside the classes.

Reply:
can this method at least have typing info on the input params.